feature/amazon/build.gradle

```diff
@@ -9,13 +9,6 @@ apply from: "$rootProject.projectDir/library.gradle"
 android {
     namespace 'com.revenuecat.purchases.amazon'
 
-    flavorDimensions "apis"
-    productFlavors {
-        defaults {
-            dimension "apis"
-        }
-    }
```

Comment:
As mentioned, the `customEntitlementComputation` flavor is needed in amazon, even though we won't be publishing it, so that the maven publish plugin can load with the `customEntitlementComputationRelease` variant name.